src/Sarif/Sarif.csproj

```diff
@@ -59,7 +59,7 @@
 
   <ItemGroup>
     <EmbeddedResource Update="SdkResources.resx">
-      <Generator>ResXFileCodeGenerator</Generator>
+      <Generator>PublicResXFileCodeGenerator</Generator>
```

Comment:
>PublicResXFileCodeGenerator [](start = 17, length = 27)

What's a PublicResXFileCodeGenerator? Is that a custom tool of ours?

Reply:
It is not a custom tool of ours. This is part of the MSBuild machinery.

When you double-click a resx file in VS, you'll see a combo box at the top that says "Access Modifier: Internal". You change "Internal" to "Public" if you want MSBuild to generate the C# class as `public` instead of `internal`. That causes the change in the project file that you see here.

Whether MSBuild has an actual tool PublicResXFileCodeGenerator, I don't know.

---
In reply to: [312230107](https://github.com/microsoft/sarif-sdk/pull/1625#discussion_r312230107) [](ancestors = 312230107)